packages/docs/src/pages/resources.mdx

```diff
@@ -0,0 +1,28 @@
+---
+name: 'Resources'
+---
+
+# Resources
+
+Tools, themes, extensions and examples built by the Theme UI community. 
+
+## Custom themes
+
+- [**@tabetalt/kit**](https://github.com/tabetalt/kit):
+  Used in Tabetalt’s e-commerce platform.
+- [**@hackclub/theme**](https://theme.hackclub.com):
+  Used on [Hack Club’s](https://hackclub.com/) websites.
+
+## Component kits
+
+- [**@component-controls/components**](https://github.com/ccontrols/component-controls/tree/master/ui/components):
+  Components built for Theme UI.
+- [**@cartolab/elements**](https://github.com/cartolab-gis/elements):
+  Themeable React components for map-based web applications.
+
+## Other projects
+
+- [**theme-ui-native**](https://github.com/samjbmason/theme-ui-native):
+  Theme UI adapted for React Native projects.
+- [**Theme UI plugin for Figma**](https://github.com/LekoArts/figma-theme-ui):
```

Comment:
I feel we should include Dripsy. It's quite awesome :)

```suggestion
- [**Dripsy**](https://github.com/nandorojo/dripsy): A responsive, theme-based design system for Expo + React Native Web.
- [**Theme UI plugin for Figma**](https://github.com/LekoArts/figma-theme-ui):
```

Reply:
Done! Hadn't heard of it, but yeah, add anything you can think of.